packages/nimble-components/src/combobox/tests/combobox.spec.ts

```diff
@@ -1,640 +1,470 @@
-    combobox.control.value = text;
-    const inputEvent = new InputEvent('input', {
-        data: text,
-        inputType: 'insertText'
-    });
-    combobox.inputHandler(inputEvent);
-    combobox.dispatchEvent(inputEvent);
-}
-
-async function clickAndWaitForOpen(combo: Combobox): Promise<void> {
-    const regionLoadedListener = createEventListener(combo, 'loaded');
-    combo.click();
-    await regionLoadedListener.promise;
-}
+import { ComboboxPageObject } from '../testing/combobox.pageobject';
 
 describe('Combobox', () => {
-    it('should set aria-disabled attribute when property is set', async () => {
-        const { element, connect, disconnect } = await setup();
-
-        element.ariaDisabled = 'true';
-        await connect();
-
-        expect(element.getAttribute('aria-disabled')).toBe('true');
-
-        await disconnect();
+    it('should export its tag', () => {
+        expect(comboboxTag).toBe('nimble-combobox');
     });
 
-    it('should set autocomplete attribute when property is set', async () => {
-        const { element, connect, disconnect } = await setup();
-
-        element.autocomplete = ComboboxAutocomplete.both;
-        await connect();
-
-        expect(element.getAttribute('autocomplete')).toBe(
-            ComboboxAutocomplete.both
+    it('can construct an element instance', () => {
+        expect(document.createElement('nimble-combobox')).toBeInstanceOf(
+            Combobox
         );
-
-        await disconnect();
     });
 
-    it('should set classes based on open, disabled, and position', async () => {
-        const { element, connect, disconnect } = await setup('above', true);
-
-        element.disabled = true;
-        await connect();
-        await waitForUpdatesAsync();
-
-        expect(element.classList.contains('above')).toBeTrue();
-        expect(element.classList.contains('disabled')).toBeTrue();
-        expect(element.classList.contains('open')).toBeTrue();
-
-        await disconnect();
-    });
-
-    it('should set tabindex based on disabled state', async () => {
-        const { element, connect, disconnect } = await setup();
-
-        await connect();
-
-        expect(element.getAttribute('tabindex')).toBe('0');
-        element.disabled = true;
-        await waitForUpdatesAsync();
-        expect(element.getAttribute('tabindex')).toBeNull();
-
-        await disconnect();
-    });
-
-    it('should forward  based on disabled state', async () => {
-        const { element, connect, disconnect } = await setup();
-
-        await connect();
-
-        expect(element.getAttribute('tabindex')).toBe('0');
-        element.disabled = true;
-        await waitForUpdatesAsync();
-        expect(element.getAttribute('tabindex')).toBeNull();
+    describe('with common setup', () => {
+        async function setup(): Promise<Fixture<Combobox>> {
+            // prettier-ignore
+            const viewTemplate = html`
+                <nimble-combobox>
```

Comment:
While we are here can you update this fixture and the others below to use the tagName imports instead of hard coding the names